src/modules/pythonmonkey/pythonmonkey.cc

```diff
@@ -608,5 +629,21 @@ PyMODINIT_FUNC PyInit_pythonmonkey(void)
     return NULL;
   }
 
+  // initialize FinalizationRegistry of JSFunctions to Python Functions
+  JS::RootedValue FinalizationRegistry(GLOBAL_CX);
+  JS::RootedObject registryObject(GLOBAL_CX);
+
+  JS_GetProperty(GLOBAL_CX, *global, "FinalizationRegistry", &FinalizationRegistry);
```

Comment:
can we use JS::SetHostCleanupFinalizationRegistryCallback for direct/more efficient access instead?

Reply:
done